plugins/in_dummy/in_dummy.c

```diff
@@ -134,12 +136,11 @@ static int configure(struct flb_dummy *ctx,
                      struct flb_input_instance *in,
                      struct timespec *tm)
 {
-    const char *str = NULL;
     struct flb_time dummy_time;
+    const char *msg;
     int dummy_time_enabled = FLB_FALSE;
     int root_type;
     int  ret = -1;
-    long val  = 0;
 
```

Comment:
Could you add `ctx->dummy_message = NULL` ?
In the future, the function may return before checking `dummy` property.

Reply:
done.